### ISSUE TITLE
Regression in block num_results in pager

### ISSUE THREAD

**Issue by alxvgt** [sonata-project/SonataAdminBundle#5700]
# Environment
Tested on linux/chrome

#### Sonata packages

```
$ composer show | grep sonata
sonata-project/admin-bundle              3.53.0     The missing Symfony Admin Generator
sonata-project/block-bundle              3.18.1     Symfony SonataBlockBundle
sonata-project/cache                     2.0.1      Cache library
sonata-project/core-bundle               3.17.0     Symfony SonataCoreBundle
sonata-project/datagrid-bundle           2.5.0      Symfony SonataDatagridBundle
sonata-project/doctrine-extensions       1.3.0      Doctrine2 behavioral extensions
sonata-project/doctrine-orm-admin-bundle 3.10.0     Symfony Sonata / Integrate Doctrine ORM into the SonataAdminBundle
sonata-project/exporter                  2.0.1      Lightweight Exporter library
sonata-project/formatter-bundle          4.1.3      Symfony SonataFormatterBundle
```

```
$ composer show --latest 'sonata-project/*'
sonata-project/admin-bundle              3.53.0 3.53.0 The missing Symfony Admin Generator
sonata-project/block-bundle              3.18.1 3.18.1 Symfony SonataBlockBundle
sonata-project/cache                     2.0.1  2.0.1  Cache library
sonata-project/core-bundle               3.17.0 3.17.0 Symfony SonataCoreBundle
sonata-project/datagrid-bundle           2.5.0  3.0.0  Symfony SonataDatagridBundle
sonata-project/doctrine-extensions       1.3.0  1.3.1  Doctrine2 behavioral extensions
sonata-project/doctrine-orm-admin-bundle 3.10.0 3.10.0 Symfony Sonata / Integrate Doctrine ORM into the SonataAdminBundle
sonata-project/exporter                  2.0.1  2.0.1  Lightweight Exporter library
sonata-project/formatter-bundle          4.1.3  4.1.3  Symfony SonataFormatterBundle

```

#### Symfony packages

```
$ composer show --latest 'symfony/*'
symfony/assetic-bundle        v2.8.2  v2.8.2  Integrates Assetic into Symfony2
Package symfony/assetic-bundle is abandoned, you should avoid using it. Use symfony/webpack-encore-pack instead.
symfony/http-client           v4.3.4  v4.3.4  Symfony HttpClient component
symfony/http-client-contracts v1.1.6  v1.1.6  Generic abstractions related to HTTP clients
symfony/mime                  v4.3.4  v4.3.4  A library to manipulate MIME messages
symfony/monolog-bundle        v3.4.0  v3.4.0  Symfony MonologBundle
symfony/phpunit-bridge        v3.4.31 v4.3.4  Symfony PHPUnit Bridge
symfony/polyfill-apcu         v1.12.0 v1.12.0 Symfony polyfill backporting apcu_* functions to lower PHP versions
symfony/polyfill-ctype        v1.12.0 v1.12.0 Symfony polyfill for ctype functions
symfony/polyfill-iconv        v1.12.0 v1.12.0 Symfony polyfill for the Iconv extension
symfony/polyfill-intl-icu     v1.12.0 v1.12.0 Symfony polyfill for intl's ICU-related data and classes
symfony/polyfill-intl-idn     v1.12.0 v1.12.0 Symfony polyfill for intl's idn_to_ascii and idn_to_utf8 functions
symfony/polyfill-mbstring     v1.12.0 v1.12.0 Symfony polyfill for the Mbstring extension
symfony/polyfill-php56        v1.12.0 v1.12.0 Symfony polyfill backporting some PHP 5.6+ features to lower PHP versions
symfony/polyfill-php70        v1.12.0 v1.12.0 Symfony polyfill backporting some PHP 7.0+ features to lower PHP versions
symfony/polyfill-php72        v1.12.0 v1.12.0 Symfony polyfill backporting some PHP 7.2+ features to lower PHP versions
symfony/polyfill-php73        v1.12.0 v1.12.0 Symfony polyfill backporting some PHP 7.3+ features to lower PHP versions
symfony/polyfill-util         v1.12.0 v1.12.0 Symfony utilities for portability of PHP codes
symfony/security-acl          v3.0.2  v3.0.2  Symfony Security Component - ACL (Access Control List)
symfony/service-contracts     v1.1.6  v1.1.6  Generic abstractions related to writing services
symfony/swiftmailer-bundle    v3.2.8  v3.2.8  Symfony SwiftmailerBundle
symfony/symfony               v3.4.31 v4.3.4  The Symfony PHP framework
symfony/webpack-encore-bundle v1.6.2  v1.6.2  Integration with your Symfony app & Webpack Encore!
```

#### PHP version

```
$ php -v
PHP 7.3.3-1+0~20190307202445.32+jessie~1.gbp32ebb2 (cli) (built: Mar  8 2019 07:03:07) ( NTS )
Copyright (c) 1997-2018 The PHP Group
Zend Engine v3.3.3, Copyright (c) 1998-2018 Zend Technologies
    with Zend OPcache v7.3.3-1+0~20190307202445.32+jessie~1.gbp32ebb2, Copyright (c) 1999-2018, by Zend Technologies
    with Xdebug v2.7.2, Copyright (c) 2002-2019, by Derick Rethans
```

## Subject

I have updated my vendors some fex days ago. Since that update, i have noticed i little changes in table footer result on dashboard list.
After some searches, it seems that the transchoice of block num_results is beconme a simple trans tag and the content is not exactly the same.
When i replace this line by the older one, all works as i expect.

## Steps to reproduce

Have a simple list dashboard with this packages versions :
```
u_web@cw-rpe-preprod7-web:/home/sites/viaeduc-v2.as2.io/web$ php composer.phar show | grep sonata
sonata-project/admin-bundle              3.52.0     The missing Symfony Admin Generator
sonata-project/block-bundle              3.17.0     Symfony SonataBlockBundle
sonata-project/cache                     2.0.1      Cache library
sonata-project/core-bundle               3.17.0     Symfony SonataCoreBundle
sonata-project/datagrid-bundle           2.5.0      Symfony SonataDatagridBundle
sonata-project/doctrine-extensions       1.3.0      Doctrine2 behavioral extensions
sonata-project/doctrine-orm-admin-bundle 3.10.0     Symfony Sonata / Integrate Doctrine ORM into the SonataAdminBundle
sonata-project/exporter                  2.0.1      Lightweight Exporter library
sonata-project/formatter-bundle          4.1.3      Symfony SonataFormatterBundle
```
- Note the table footer with results count that's look correct
- Run a composer update
- The table footer display weird informations with an unexpected ("1 result" ...)

## Expected results

The same display as the previous version
Like that 
![image](https://user-images.githubusercontent.com/1221867/65979955-89c25900-e476-11e9-9aa2-6bcdf1a834ae.png)

## Actual results

![image](https://user-images.githubusercontent.com/1221867/65980063-b8403400-e476-11e9-9177-91244af94dba.png)

## Debug

To test the regression, i have made a little test in `vendor/sonata-project/admin-bundle/src/Resources/views/Pager/base_results.html.twig`
With the new line commented and the old line uncommented, all its ok.

```
{% block num_results %}
{#    {% trans with {'%count%': admin.datagrid.pager.nbresults} from 'SonataAdminBundle' %}list_results_count{% endtrans %}#}
    {% transchoice admin.datagrid.pager.nbresults with {'%count%': admin.datagrid.pager.nbresults} from 'SonataAdminBundle' %}list_results_count{% endtranschoice %}
    &nbsp;-&nbsp;
{% endblock %}
```

**Comment by phansys** [comment]
This is a duplicate of #5686, which is already fixed at #5689.
There is no stable release shipping this fix yet, so probably you may want to blacklist "sonata-project/admin-bundle:3.53.0" from your dependencies.

/cc @sonata-project/release-managers.

**Comment by alxvgt** [comment]
Thank you for your very quick answer !
You released a the new 3.54.0, i will run a composer update now ;-)
